### ISSUE TITLE
Can RoleSessionName be made configurable?

### ISSUE THREAD

**Issue by james-hu** [jamesmbourne/aws4-axios#1127]
Thank you for making this package, it is quite handy!
I just want to propose making RoleSessionName configurable. This is the informaiton passed from the caller through IAM to the service.

It seems quite easy to mak it configurable. Basically just need to change
```
      new AssumeRoleCommand({
        RoleArn: this.options.roleArn,
        RoleSessionName: "axios",
      })
```
to
```
      new AssumeRoleCommand({
        RoleArn: this.options.roleArn,
        RoleSessionName: this.options.roleSessionName || "axios",
      })
```

**Comment by jamesmbourne** [comment]
Hey @james-hu, thanks for the suggestion! If you're able to create a PR for this I'd be happy to review it.

**Comment by jamesmbourne** [comment]
This should be released in v3.3.0, thanks for the contribution @james-hu!

**Comment by jamesmbourne** [comment]
@all-contributors please add @james-hu for code
